### PR TITLE
Googleslides: Updating the workload to work with version 1.18 and sup…

### PR DESCRIPTION
**Pull request by scojac01**
…port the latest Huawei devices

### CODE REVIEW

File: wa/workloads/googleslides/__init__.py [hunk 1]
```diff
@@ -121,3 +121,19 @@ def init_resources(self, context):
...
+
+    def deploy_assets(self, context):
+        super(GoogleSlides, self).deploy_assets(context)
+        #Move the test files to the download folder
+        d = self.target.working_directory
+        e = self.target.external_storage
+
+        orig_file_path = self.target.path.join(d, self.test_file)
+        new_file_path = self.target.path.join(e, 'Download')
+        self.target.execute('cp {} {}'.format(orig_file_path, new_file_path))
```
Comment (marcbonnici) [inline wa/workloads/googleslides/__init__.py]: Might be worth adding a `self.target.refresh_files(self.deployed_assets)` here?

File: wa/workloads/googleslides/__init__.py [hunk 2]
```diff
@@ -121,3 +121,19 @@ def init_resources(self, context):
...
             raise ValidationError('{} must be a PPTX file'.format(self.test_file))
+
+    def deploy_assets(self, context):
+        super(GoogleSlides, self).deploy_assets(context)
+        #Move the test files to the download folder
+        d = self.target.working_directory
+        e = self.target.external_storage
+
+        orig_file_path = self.target.path.join(d, self.test_file)
+        new_file_path = self.target.path.join(e, 'Download')
```
Comment (marcbonnici) [inline wa/workloads/googleslides/__init__.py]: Need to add the `new_file_path` to `self.deployed_assets` for the clean up to work correctly.

**Comment by scojac01** [comment]
Updated.

File: wa/workloads/googleslides/__init__.py [hunk 3]
```diff
@@ -121,3 +121,21 @@ def init_resources(self, context):
...
```
Comment (marcbonnici) [inline wa/workloads/googleslides/__init__.py]: Just checking that you meant the 'Download' folder as opposed to the 'Downloads' folder? 
And also will this always be present on the device or should there be a check / creation of the folder?

File: wa/workloads/googleslides/__init__.py [hunk 4]
```diff
@@ -121,3 +121,21 @@ def init_resources(self, context):
...
+        self.deployed_assets.append(new_file_path)
+        self.target.execute('cp {} {}'.format(orig_file_path, new_file_path))
+        self.target.refresh_files(self.deployed_assets)
+
+    def remove_assets(self, context):
+        for asset in self.deployed_assets:
+            self.target.remove(os.path.dirname(asset))
+        self.target.refresh_files(self.deployed_assets)
```
Comment (marcbonnici) [inline wa/workloads/googleslides/__init__.py]: Is your intention to to delete the entire 'Download' folder here? 
If not then this method can be removed and the built-in functionality will remove the individual files.

File: wa/workloads/googleslides/__init__.py (hunk 3 above)
Comment (scojac01) [reply wa/workloads/googleslides/__init__.py]: Downloads is correct. As far as I can tell, this is the default folder on all devices.

File: wa/workloads/googleslides/__init__.py (hunk 4 above)
Comment (scojac01) [reply wa/workloads/googleslides/__init__.py]: That is not my intention. I will remove.

File: wa/workloads/googleslides/__init__.py (hunk 2 above)
Comment (scojac01) [reply wa/workloads/googleslides/__init__.py]: Isn't this happening on line 133? 

self.deployed_assets.append(new_file_path)

**Pull request closed without merging**